src/contributing/release_checklist.md

```diff
@@ -0,0 +1,54 @@
+# The Parsec Release Checklist
+
+The release process is divided into 4 stages as shown below. Each stage has a discrete set of
+activities to be completed before progressing to the next stage. The list shown below is only a
+template, as there might be additional tasks specific to that release that might be added to
+different stages.
+
+Stages:
+
+- Prior Code freeze
+   - [ ]  Publish new versions of all managed crates and configure parsec service and tool to use
+      them. See
+      this [guide](https://parallaxsecond.github.io/parsec-book/contributing/package_management.html).
+   - [ ]  Fix any pending issues
+   - [ ]  Check common dependency crates versions between the parsec service, parsec tool, and
+      other crates
+   - [ ]  Review PRs across Parsec repos
+   - [ ]  Merge PRs got approved
+   - [ ]  Check the milestone list
+- During code freeze
+   - [ ]  Issue release candidate tag x.x.x-rc1
+   - [ ]  OpenSUSE packaging for the RC
+   - [ ]  Pull in the latest git dependencies in the crates to ensure the latest release would
+      work.
+   - [ ]  Execute the normal e2e tests on Raspberry Pi hardware:
+      - mbed-crypto
+      - TPM
+      - Nitrokey HSM (pluggable USB)
+   - [ ]  Execute fuzz testing for some time
```

Comment:
Could we maybe expand on the "for some time"?

Reply:
Yes, will update it.